### PR TITLE
Update attack-simulation-training-training-campaigns.md

### LINKED ISSUE #12372: Type of Groups that are eligible
Please specify what type of Groups can be assigned to a training campaign, I am not seeing Security Groups, so it appears that only M365 Groups are eligible. It would be helpful to explicitly state this . 

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: e368fd8c-e40c-7999-a151-36bbb13ba52d
* Version Independent ID: e368fd8c-e40c-7999-a151-36bbb13ba52d
* Content: [Training campaigns in Attack simulation training](https://learn.microsoft.com/en-us/microsoft-365/security/office-365-security/attack-simulation-training-training-campaigns?view=o365-worldwide)
* Content Source: [microsoft-365/security/office-365-security/attack-simulation-training-training-campaigns.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/attack-simulation-training-training-campaigns.md)
* Service: **microsoft-365-security**
* Sub-service: **mdo**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about the groups requirement.

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12372

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/office-365-security/attack-simulation-training-training-campaigns.md [hunk 1]
```diff
@@ -106,6 +106,8 @@ On the **Target users** page, select who receives the Training campaign. Use the
     - Type fewer than three characters or no characters and then press the ENTER key. No users are shown in the **User list** section, but you can type three or more characters in the **Search** box to search for users and groups.
 
     The number of results appears in the **Selected (0/x) users** label.
+    > [!NOTE]
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/attack-simulation-training-training-campaigns.md:109]:
```suggestion
 
       > [!NOTE]
```

Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/attack-simulation-training-training-campaigns.md]:
```suggestion

    > [!NOTE]
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**